src/clojure/yaw/reaction.clj

```diff
@@ -50,3 +50,61 @@
                                        ;; (println "Update! " delta-time " ms")
                                        (swap! ratom (fn [_] delta-time)))))
     ratom))
+
+(defonce app-db (atom {}))
```

Comment:
A common although not always enforced convention is to write global variables in a special way. The convention I used until today is `+name-of-the-variable+`.  That's not beautiful but then we know it's something special (`*name-of-the-variable*` can only be used for *dynamic vars*).